File: src/main/java-graphql/io/github/graphql/model/UserContributionsCollectionParametrizedInput.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLParametrizedInput;
import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLRequestSerializer;
import java.util.StringJoiner;

/**
 * Parametrized input for field contributionsCollection in type User
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public class UserContributionsCollectionParametrizedInput implements GraphQLParametrizedInput {

    private String from;
    private String organizationID;
    private String to;

    public UserContributionsCollectionParametrizedInput() {
    }

    public UserContributionsCollectionParametrizedInput(String from, String organizationID, String to) {
        this.from = from;
        this.organizationID = organizationID;
        this.to = to;
    }

    public UserContributionsCollectionParametrizedInput from(String from) {
        this.from = from;
        return this;
    }

    public UserContributionsCollectionParametrizedInput organizationID(String organizationID) {
        this.organizationID = organizationID;
        return this;
    }

    public UserContributionsCollectionParametrizedInput to(String to) {
        this.to = to;
        return this;
    }


    @Override
    public String toString() {
        StringJoiner joiner = new StringJoiner(", ", "(", ")");
        if (from != null) {
            joiner.add("from: " + GraphQLRequestSerializer.getEntry(from));
        }
        if (organizationID != null) {
            joiner.add("organizationID: " + GraphQLRequestSerializer.getEntry(organizationID));
        }
        if (to != null) {
            joiner.add("to: " + GraphQLRequestSerializer.getEntry(to));
        }
        return joiner.toString();
    }

}

File: src/main/kotlin-graphql/io/github/graphql/model/UserContributionsCollectionParametrizedInput.kt
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLParametrizedInput
import java.util.StringJoiner
/**
 * Parametrized input for field contributionsCollection in type User
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-06T19:48:04+0800"
)
data class UserContributionsCollectionParametrizedInput(
    val from: String?,
    val organizationID: String?,
    val to: String?
) : GraphQLParametrizedInput {

    override fun toString(): String {
        val joiner = StringJoiner(", ", "( ", " )")
        if (from != null) {
            joiner.add("from: " + GraphQLRequestSerializer.getEntry(from))
        }
        if (organizationID != null) {
            joiner.add("organizationID: " + GraphQLRequestSerializer.getEntry(organizationID))
        }
        if (to != null) {
            joiner.add("to: " + GraphQLRequestSerializer.getEntry(to))
        }
        return joiner.toString()
    }
}
